components/Page.js

```diff
@@ -24,6 +25,10 @@ class Page extends Component {
 
 export default Page;
 
+Page.propTypes = {
+  children: PropTypes.object,
```

Comment:
Looks like we're expecting `children` to [be one element right now](https://github.com/texas-justice-initiative/website-nextjs/blob/369fcb1d17f7ce9bbf012ab02b2363af2e104357/pages/_app.js#L22-L26). What do you think about using the doc's suggestion about [requiring a single child](https://reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child)?

```suggestion
  children: PropTypes.element.isRequired,
```

If we think we'll want to accept multiple children, [StackOverflow](https://stackoverflow.com/questions/42122522/reactjs-what-should-the-proptypes-be-for-this-props-children) seems to prefer `PropTypes.node.isRequired`.